zproject/prod_settings_template.py

```diff
@@ -435,7 +435,7 @@
         ## to be a SAML attribute containing the allowed subdomains for a user.
         # "attr_org_membership": "member",
     },
-}
+}  # We want this to use TypedDict so we can't let mypy interpolate the type here.
```

Comment:
I'm going to remove this line since I think the benefit of the comment is outweighed by the confusion for sysadmins reading this of having it here.